src/transformers/models/stablelm/modeling_stablelm.py

```diff
@@ -374,6 +374,82 @@ def forward(
         return attn_output, attn_weights, past_key_value
 
 
+class StableLmSdpaAttention(StableLmAttention):
+    def forward(
+        self,
+        hidden_states: torch.Tensor,
+        attention_mask: Optional[torch.Tensor] = None,
+        position_ids: Optional[torch.LongTensor] = None,
+        past_key_value: Optional[Cache] = None,
+        output_attentions: bool = False,
+        use_cache: bool = False,
+    ) -> Tuple[torch.Tensor, Optional[torch.Tensor], Optional[Tuple[torch.Tensor]]]:
+        bsz, q_len, _ = hidden_states.size()
```

Comment:
BTW we need the logic to call `super().forward` unless all the tests run locally. see this: 

https://github.com/huggingface/transformers/blob/main/src/transformers/models/llama/modeling_llama.py#L597